### ISSUE TITLE
BakedQuad Fix Problems

### ISSUE THREAD

**Issue by voidsong-dragonfly** [LoliKingdom/LoliASM#32]
1) LoliASM adding its own classes to BakedQuad override -  it's adding `zone.rong.loliasm.bakedquad.SupportingBakedQuad` to the config list. This then causes crashes with IE (and probably some other mods too) due to NoClassDefFound errors. Pretty much everything works fine without the insertion. Log's [here](https://gist.github.com/Silfryi/3150a262fc5b45ab8e7cb0edea9c05b6)
2) When it modifies my IW fork's BakedQuad stuff, the flywheels no longer render their outer parts. I'm not 100% sure if this is because of the modifications LoliASM does [to this class](https://github.com/Silfryi/IndustrialWires/blob/MC1.12/src/main/java/malte0811/industrialwires/client/render/TileRenderMechMB.java), or if it's due to how I grab textures from blocks [here (different class)](https://github.com/Silfryi/IndustrialWires/blob/6720f0135793a0f1bc5837cfacddf396d2edd004/src/main/java/malte0811/industrialwires/mech_mb/Material.java#L88) but something is causing them not to render.
3) Grass is rendering as its default grey color, non-biome-color mapped. Unsure which mod of mine is doing it, if it is one, but removing just BiomeTweaker + BTCore nor just Quark fixed this.
4) LoliASM with the config linked [here](https://gist.github.com/Silfryi/a27a0b93e1e225bceb4792043fd1f549) crashes with CTM with [this](https://gist.github.com/Silfryi/ca39e5716f6c867453b34c7bfe12239f) error log
5) Item frames seem to cause the server to simply hang. This was after I disabled Quark to see if it was the single cause of the grass (no), so I don't know if they would with Quark. Nothing provided for a crash report, game simply hung and needed to be killed.

Any problem was tested without 4) except for 4) due to the game not loading with CTM enabled. Config provided for 4) was common across all issues except for 1), which has its note on what LoliASM added.
Neither 2) nor 3) seem to leave anything in the log at the time of the problem. Log has been provided for those two [here](https://gist.github.com/Silfryi/04da58a6626b11ec29d168f901733645) in case you want it

[Modlist is here](https://gist.github.com/Silfryi/fe5b4411841eeb7a5dca1055bc21b6fb) - All issues were tested with this modlist unless otherwise specified

**Comment by Rongmario** [comment]
I'll take a look, thanks for the info provided.

**Comment by Silfryi** [comment]
Np! If you want any more just ask. I tried to provide as much as I could because the bugs were pretty elusive and having too little info is never fun.

**Comment by SilverBackSpartan** [comment]
verified
[crash-2021-10-17_04.30.55-client.txt](https://github.com/LoliKingdom/LoliASM/files/7359472/crash-2021-10-17_04.30.55-client.txt)

**Comment by Silfryi** [comment]
Update: #1 and #4 seem fixed with 3.2.1. I now actually have a crash for the item frame thing, which is gotten with the above config/modlist/etc on 3.2.1. Report is [here](https://gist.github.com/Silfryi/90def879e705ec8b4407b1920c9b6c78)

**Comment by Rongmario** [comment]
@SilverBackSpartan that'll be fixed in 3.3, just need to handle heuristic edge-case

**Comment by Silfryi** [comment]
@Rongmario here's the new log upon rendering the item frame and crashing: [log](https://gist.github.com/Silfryi/33b0b9868a868bc92443c58250330ab5)

**Comment by Rongmario** [comment]
@Silfryi mind just testing 4.4.1 when it gets released on CF? Just so we can try consolidate and find what the status is of this problem now.

**Comment by Silfryi** [comment]
Yeah, will do. I've mostly moved on to 1.16 atm but it should be no issue.

**Comment by Rongmario** [comment]
What a shame :'(

**Comment by Silfryi** [comment]
AR's big enough I can only support one version, and the one I need to support is 1.16. I'll still test, I just won't be doing large amounts of playtesting to see if it's fixed fixed,

**Comment by Rongmario** [comment]
It's fine, np

**Comment by Silfryi** [comment]
Just tested, and 4.4.1 does not seem to fix # 2, which is the last problem still standing for this issue. Unsure what the cause is, but the quads still are not rendering.
